src/services/leafletLayerHelpers.js

```diff
@@ -55,6 +55,14 @@ angular.module('ui-leaflet')
                 return L.tileLayer(url, params.options);
             }
         },
```

Comment:
Yeah, was running into that issue myself ... was looking for easiest change to ui-leaflet to facilitate